networking_oneview/ml2/drivers/oneview/neutron_oneview_client.py

```diff
@@ -183,6 +182,18 @@ def _create_network_on_oneview(self, name, network_type, seg_id):
         }
         return self.oneview_client.ethernet_networks.create(options)
 
+    def _add_to_ligs(self, network_type, physical_network, oneview_net_uri):
+        try:
```

Comment:
Maybe, if this method didn't have a try...except clause, we didn't need to worry about raising an Exception. The Exception would be raised by one of the methods in here.

Reply:
If so then I'm totally into changing to it!